src/index.html

```diff
@@ -1,12 +1,354 @@
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__product-review">Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <span class="card__price-text">Price:</span>
+              <span class="card__product-price">$2,199</span>
+            </div>
+
+            <a href="#" class="card__link">
+              <button type="submit" class="card__buy">Buy</button>
+            </a>
+          </div>
+        </section>
+
+        <section class="card">
+          <img 
+            class="card__image" 
+            src="./images/imac.jpeg" 
+            alt="imac"
+          >
+
+          <div class="card__section">
+            <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <div class="card__product-number">Product code: 195434</div>
+
+            <div class="card__stars-and-review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__product-review">Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <span class="card__price-text">Price:</span>
+              <span class="card__product-price">$2,199</span>
+            </div>
+
+            <a href="#" class="card__link">
+              <button type="submit" class="card__buy">Buy</button>
+            </a>
+          </div>
+        </section>
+
+        <section class="card">
+          <img 
+            class="card__image" 
+            src="./images/imac.jpeg" 
+            alt="imac"
+          >
+
+          <div class="card__section">
+            <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <div class="card__product-number">Product code: 195434</div>
+
+            <div class="card__stars-and-review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__product-review">Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <span class="card__price-text">Price:</span>
+              <span class="card__product-price">$2,199</span>
+            </div>
+
+            <a href="#" class="card__link">
+              <button type="submit" class="card__buy">Buy</button>
+            </a>
+          </div>
+        </section>
+      </div>
+
+    </main>
+
+
   </body>
-</html>
+
+</html>
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)